src/Processor/LastModifiedProcessor.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+/*
+ * This file is part of the "Tom32i/Content" bundle.
+ *
+ * @author Thomas Jarrand <thomas.jarrand@gmail.com>
+ */
+
+namespace Content\Processor;
+
+use Content\Behaviour\ProcessorInterface;
+use Content\Content;
+
+/**
+ * Set a "LastModified" property based on file date
+ */
+class LastModifiedProcessor implements ProcessorInterface
+{
+    public static function isSupported($value): bool
+    {
+        return \is_null($value);
+    }
+
+    public function __invoke(array &$data, array $context): void
+    {
+        if (!static::isSupported($data['lastModified'] ?? null)) {
```

Comment:
```suggestion
        if (!isset($data['lastModified'])) {
```

(en fait pour le coup ça revient à faire un isset, le null coalescing est surtout cool pour stocker la valeur en même temps que la condition pour la ré-exploiter ensuite. ex: `if ($foo = $array['foo'] ?? false`) { echo $foo }`